### DISCUSSION TITLE
文章开头一段格式段落全都消失！

### DISCUSSION THREAD

**Discussion by AttheHawksWell** [hexojs/hexo#5272] [Question / Answer]
![QQ截图20230820054157](https://github.com/hexojs/hexo/assets/117459605/936abe00-4143-4710-bc9c-7f899ddaaee0)
![QQ截图20230820054118](https://github.com/hexojs/hexo/assets/117459605/57186937-ae38-4cb9-bb22-61365ba6f010)
明明在Typora里预览的格式就是正常的，但上传到博客内文章开头一段就格式全都消失也不知道为什么…………怀疑过是字数太多的问题已经尝试将文章拆分成二份分开发布但是依然没有起色。
本人为小白，基本照着他人的教程搭建博客，可否有大神解答一下这种问题如何解决T T已善用搜索引擎

**Comment by stevenjoezhang** [comment]
你可以提供一下你的网站的链接

**Comment by uiolee** [comment]
maybe it's a known issue

hexojs/hexo#5309
next-theme/hexo-theme-next#707
